libpod/container_internal_linux.go

```diff
@@ -87,6 +87,11 @@ func (c *Container) prepare() (err error) {
 				c.state.NetNS = netNS
 				c.state.NetworkStatus = networkStatus
 			}
+
+			// Setup rootless networking, requires c.state.NetNS to be set
+			if rootless.IsRootless() {
+				rootlessSetupErr = c.runtime.setupRootlessNetNS(c)
```

Comment:
Question: why isn't this fatal? Are we expecting a lot of spurious errors here? I would expect a slirp failure to prevent container creation

Reply:
Because it is a go routine, it it fatal once the other go routine completes, and the error is checked.